apps/qubit/modules/object/actions/importAction.class.php

```diff
@@ -51,103 +51,30 @@ public function execute($request)
       $this->redirect($importSelectRoute);
     }
 
-    // set indexing preference
-    if (isset($request->noindex))
+    $options = array('noIndex' => ($request->getParameter('noindex') == 'on') ? false : true,
+                     'doCsvTransform' => ($request->getParameter('doCsvTransform') == 'on') ? true : false,
```

Comment:
I wrote it this way for clarity, as the arFileImportJob depends on it being true/false, as opposed to 'on' or ''.  I don't think there is a technical reason preventing me from shortening it.  Thoughts?